demo/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+# Copyright 2022 The Chromium Authors.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+cmake_minimum_required(VERSION 3.22)
+
+project(chrome_enterprise_connector_local_analysis)
+
+# Makes available the package definitions in vcpkg.
+include(build/vcpkg/scripts/buildsystems/vcpkg.cmake)
```

Comment:
Why is this line needed?  It also seems to hard code `build/...` as the directory where the demo is built.  Can you try specifying a directory outside the source tree to make that works too?

If this line is needed I think you need to replace`build` with `${PROJECT_BINARY_DIR}`